backend/main/workers/tracebacks.py

```diff
@@ -32,6 +31,12 @@
 
 log = logging.getLogger(__name__)
 
+def get_friendly_object(e):
+    try:
+        fr = FriendlyTraceback(type(e), e, e.__traceback__)
+        return fr
+    except (Exception, SystemExit):
+        log.exception("Failed to build FriendlyTraceback object")
```

Comment:
This returns None and then causes another exception later. Don't call the other friendly functions when this happens.